wiki/Tournaments/OFFC/cirno/en.md

```diff
@@ -0,0 +1,225 @@
+---
+tags:
+  - OFFC cirno
+  - cirno
+  - "OFFC: cirno"
+  - OFFC
+---
+
+# osu! FumoFumo Cup: cirno
+
+![OFFC: cirno logo](img/logo.png)
+
+The **osu! FumoFumo Cup: cirno** (***OFFC: cirno***) is a 4v4 double-elimination osu!standard tournament hosted by ::{ flag=CN }:: [kitakore](https://osu.ppy.sh/users/33887163), which is also the inaugural season of OFFC.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2024-08-10/2024-08-30 |
+| Team-up phase | 2024-08-20/2024-09-06 |
+| Quarterfinals | 2024-09-09/2024-09-17 |
+| Semifinals | 2024-09-18/2024-09-30 |
+| Finals | 2024-10-01/2024-10-07 |
+| Grand finals | 2024-10-08/2024-10-20 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | One MeowPad for each team member (provided by *DESU.Life*) |
```

Comment:
might want to add a link to MeowPad here, as it's not really clear to the readers what a "MeowPad" is on the first glance

something like this :

![image](https://github.com/user-attachments/assets/071cff47-45a1-4e2f-8947-c3642df2d6c1)